lib/jsdom/browser/resource-loader.js

```diff
@@ -170,7 +200,7 @@ exports.download = function (url, options, cookieJar, referrer, callback) {
   };
 };
 
-exports.load = function (element, url, callback, encoding) {
+exports.load = function (element, url, callback, encoding, detectMetaCharset) {
```

Comment:
Can you use an options object instead of a boolean parameter? See http://ariya.ofilabs.com/2011/08/hall-of-api-shame-boolean-trap.html.

If you'd like to reorganize this while you're here, I think the best signatures would be

- download(url, options, callback) with options containing all the things it contains now plus cookieJar, referrer, and detectMetacharset
- load(element, url, options, callback) with options containing encoding and detectMetaCharset